ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -20,12 +20,43 @@
 from abc import ABC, abstractmethod
 from typing import List, NamedTuple, Tuple, Optional, Union, Dict
 import numpy as np
-from enum import Enum
+from enum import Enum, IntEnum
 
 AgentId = int
 AgentGroup = str
 
 
+class EpisodeStatus(IntEnum):
+    """
+    The Episode is still ongoing
+    """
+
+    Default = 0
+
+    """
+    The Episode ended. The Agent either failed or succeeded at its task.
+    """
+    Terminated = 1
+
+    """
+    The Episode was interrupted. The Agent neither failed nor succeeded at the
+    task since it was interrupted.
+    """
+    Interrupted = 2
+
+    def is_done(self) -> bool:
+        """
+        return True if the episode was terminated or interrupted
+        """
+        return self.value == 1 or self.value == 2  # pylint: disable=W0143
```

Comment:
nit: use the enum here, not the int. pretty sure `EpisodeStatus.Terminated` will work here; if not, I think `self.Terminated` will.